tests/automated_tests/test.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
This test is testing only runtimes. 
Please rename it or make it more obvious that it is test_suit and will be extended in the future

Reply:
Actually, this is not an integration test, this is an automated test which ensures that template with all dependencies works as expected.